CentralComputing/Pod.cpp

```diff
@@ -26,7 +27,7 @@ void Pod::logic_loop() {
       command->value = 0;
     }
 
-    Utils::toggle_GPIO_high(HEARTBEAT_GPIO, switchVal);
+    Utils::set_GPIO(HEARTBEAT_GPIO, switchVal);
```

Comment:
I changed this to `set_GPIO` to be pedantic. We are not toggling since we are giving the function the value we want to *set*